src/main/java/org/folio/inventory/resources/MoveApi.java

```diff
@@ -65,17 +79,18 @@ private void moveItems(RoutingContext routingContext) {
           if (Objects.nonNull(holding)) {
             try {
               OkapiHttpClient okapiClient = createHttpClient(routingContext, context);
-              CollectionResourceClient itemsStorageClient = createItemsStorageClient(okapiClient, context);
-              MultipleRecordsFetchClient multipleRecordsFetchClient = createItemsFetchClient(itemsStorageClient);
+              CollectionResourceClient itemsStorageClient = createStorageClient(okapiClient, context, ITEM_STORAGE);
```

Comment:
I'd move it to a separate `createItemStorageClient/createHoldingsStorageClient`

Reply:
If we create two separate methods for item and storage clients then they will differ only in collection property name. The rest of the logic will be duplicated. Do you think it makes sense to do this? createStorageClient-method is private and going to be used only in scope of this class.

Reply:
we can have the common logic in parameterized `createStorageClient` and then have a short hand methods to create a client for items/holdings which will be just delegates with proper arguments. 

With this approach we can improve readability.